src/components/pages/navbar/Profile.js

```diff
@@ -0,0 +1,67 @@
+import { BsPersonCircle } from 'react-icons/bs';
+import { Link } from 'react-router-dom';
+import styled from 'styled-components';
+import { theme } from '../../../theme';
+
+export default function Profile({ name }) {
+  return (
+    <ProfileStyled>
+      <div className="user-info font_open-sans">
```

Comment:
```suggestion
      <div className="user-info">
```

À quoi sert "font_open-sans" ? 🤔